presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -15,44 +28,63 @@ The driver is also available from Maven Central:
         <version>\ |version|\ </version>
     </dependency>
 
-Requirements
-------------
+You should be using the same version of the JDBC driver as your Presto
+installation. If you are not on the latest release shown above, a list of all
+currently available versions can be found `here <https://repo1.maven.org/maven2/io/prestosql/presto-jdbc>`__.
```

Comment:
This page is not the nicest one. Maybe we link to https://search.maven.org/artifact/io.prestosql/presto-jdbc instead?